lua/lsp/null-ls.lua

```diff
@@ -24,10 +24,14 @@ function M.get_registered_providers_by_filetype(ft)
 end
 
 local function validate_nodejs_provider(requests, provider)
-  vim.cmd "let root_dir = FindRootDirectory()"
-  local root_dir = vim.api.nvim_get_var "root_dir"
+  local ts_client = require("utils").get_active_client_by_ft "typescript"
```

Comment:
I'm using prettier for markdown files, and this breaks that 
![unknown](https://user-images.githubusercontent.com/10992695/128079010-298f145c-c543-4b97-a138-f4eb5b063452.png)

Reply:
resolved in 6c6fb67a8835cdcf911ef48ec2c19d553fe7d1a1